flaml/automl/model.py

```diff
@@ -837,8 +837,11 @@ def predict_proba(self, X, **pred_kwargs):
         test_dataset = Dataset.from_pandas(X_test)
 
         new_trainer = self._init_model_for_predict()
-        predictions = new_trainer.predict(test_dataset)
-        return predictions.predictions
+        try:
+            predictions = new_trainer.predict(test_dataset)
+            return predictions.predictions
+        except ZeroDivisionError:
+            return []

```

Comment:
Can the consumer of predictions (i.e., the evaluator) handle the empty list as returned value? I doubt it.

Reply:
previously I tried to return None and now with the return []. If neither works, do you have any recommendations on how to handle the ZeroDivisionError? Maybe print a warning and return None?